src/app/input/input.component.html

```diff
@@ -1,4 +1,7 @@
 <mat-card *ngIf="service.data$ | async as data">
   <mat-card-title>Input</mat-card-title>
-  <textarea matInput class="inputTextBox" [(ngModel)]="data.inputString"></textarea>
+  <ngx-codemirror class="inputTextBox" [(ngModel)]="data.inputString" [options]="{
```

Comment:
Can you put this in an "optionsConfig" in the component itself and then reference it here.   Although the list is quite small now,  it will be more flexible and manageable if you decide to add more items later.  Change here as well as in the output component.